canvasData.py

```diff
@@ -35,18 +40,124 @@
 canvas = Canvas(CANVAS_API_URL, CANVAS_API_TOKEN)
 
 
+class CanvasUser(User):
+    id: int
+    name: str
+    sortable_name: str
+    login_id: str
+
+
 class CanvasCourse(Course):
     id: int
     name: str
+    course_code: str
 
 
 class CanvasAssignment(Assignment):
     id: int
     name: str
-    description: str
+    course_id: int
     peer_reviews: bool
     rubric_settings: dict
 
 
+class CanvasComment(object):
+    def __init__(self, comment: dict):
+        self.__comment = comment
+
+    @property
+    def criterionId(self) -> int:
+        return int(self.__comment['criterion_id'][1:])
+
+    @property
+    def comments(self) -> str:
+        return self.__comment['comments']
+
+
+class CanvasAssessment(object):
+    def __init__(self, assessment: dict):
+        self.__assessment = assessment
+
+    @property
+    def id(self) -> int:
+        return self.__assessment['id']
+
+    @property
+    def assessorId(self) -> int:
+        return self.__assessment['assessor_id']
+
+    @property
+    def isPeerReview(self) -> bool:
+        return self.__assessment['assessment_type'] == 'peer_review'
+
+    @property
+    def hasSubmission(self) -> bool:
+        return self.__assessment['artifact_type'] == 'Submission'
+
+    @property
+    def submissionId(self) -> int:
+        return self.__assessment['artifact_id'] if self.hasSubmission else None
+
+    @property
+    def comments(self) -> List[CanvasComment]:
+        return self.__assessment['data']
+
+    data: List[dict]
+
+
+class CanvasCriteria(object):
+    def __init__(self, criteria: dict):
+        self.__criteria = criteria
+
+    @property
+    def id(self) -> int:
+        '''
+        Canvas' RubricCriterion objects contain ID *strings* of the format
```

Comment:
interesting observation. We may bring this up in the upcoming meeting with Instructure.

Reply:
Great idea!